Zend/tests/function_arguments/sensitive_parameter.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
+The SensitiveParameter attribute suppresses the single sensitive argument.
+--FILE--
+<?php
+
+function test(#[SensitiveParameter] $sensitive)
+{
+    debug_print_backtrace();
+    var_dump(debug_backtrace());
+    var_dump((new Exception)->getTrace());
+}
+
+test('sensitive');
+
```

Comment:
Please add `?>` to all your tests, this makes them executable directly and eases debugging.

Reply:
Interesting. Didn't think of this use case, that would certainly have simplified some tests, thanks.